src/utils/EventEmitter.spec.ts

```diff
@@ -19,7 +19,7 @@ import {EventEmitter} from './EventEmitter.js';
 import {expect} from 'chai';
 import sinon from 'sinon';
 
-describe('EventEmitter', () => {
+describe('test EventEmitter', () => {
```

Comment:
is the `test` needed? Looks redundant

Reply:
it looks like it is written everywhere, wdyt about removing it everywhere?

Reply:
It is redundant, I just chose the style that was already predominant ('test' everywhere) and made everything consistent.

I'll remove all "test" then.